up42/auth.py

```diff
@@ -100,41 +123,55 @@ def __init__(
         except KeyError:
             self.authenticate = True
 
+        if all([arg in kwargs for arg in ["project_id", "project_api_key"]]) and auth_type == AuthType.PROJECT.value:
+            self.credentials_id = kwargs["project_id"]
+            self.credentials_key = kwargs["project_api_key"]
+            self.project_id = self.credentials_id
+
         if self.authenticate:
             self._find_credentials()
             self._get_token()
-            self._get_workspace()
+            self._get_user_id()
             logger.info("Authentication with UP42 successful!")
 
     def __repr__(self):
```

Comment:
Not needed at all

Reply:
Unaddressed

Reply:
I am not sure about this comment. There are still some endpoints that require to add workspace_id or user_id (e.g. order endpoints V1, which will be soon migrated). I think that we still need workspace_id while we finish to migrate V1 endpoints. Let me know, what do you mean by not needed at all.

Reply:
clear.